### DISCUSSION TITLE
مشکل با ssl سایت های ایرانی

### DISCUSSION THREAD

**Discussion by salaryy** [hiddify/Hiddify-Manager#555] [Q&A سوالات رایج]
سلام دوستان
 کسی با ssl سایت های ایرانی مشکل نداره ؟
 همشون ارورPR_END_OF_FILE_ERROR دارن و باز نمیشن
 من فکر میکنم باگ داره 
![image](https://user-images.githubusercontent.com/26680104/229302648-c4ca39ee-a757-47fc-b238-fbec7a59c6a9.png)
hiddify

**Comment by hntcAir** [comment]
سایت‌های ایرانی رو بلاک کردین به صورت پیشفرض روی پنل همینه.

میتونین برید تیکش رو بردارید و تغییرات اعمال کنین.

**Comment by fatehiman** [comment]
تیکش کجا هست؟ پیدا نمیکنم. میشه بگین؟
